packages/dds/cell/src/cell.ts

```diff
@@ -240,9 +222,9 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.loadCore}
      */
     protected async loadCore(storage: IChannelStorageService): Promise<void> {
-        const content = await readAndParse<ICellValue>(storage, snapshotFileName);
+        const content = await readAndParse<any>(storage, snapshotFileName);
 
-        this.data = this.fromSerializable(content);
+        this.data = this.serializer.decode(content);
```

Comment:
This will not be backward compatible for data at rest right? Are we explicitly breaking it for SharedCell? (because no one uses it? how can we sure?) @DLehenbauer @vladsud what are you thoughts?

Reply:
Scarlett grepped the repos we are aware of to bolster our confidence that SharedCell is unused.

The motivation for the change is that the legacy code in SharedCell has caused confusion in the past for developers using it as a sample/template for learning about or writing a DDS.